scripts/release-ckeditor5.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+# @license Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+# For licensing, see LICENSE.md.
+
+set -e
+
+read -p "Did you update the README.md? " -n 1 -r
```

Comment:
With what? Only if you'll reply "no" you can now learn what you need to update :D